src/test/kotlin/no/nav/syfo/infrastructure/database/TestDatabase.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.syfo.infrastructure.database
+
+import io.zonky.test.db.postgres.embedded.EmbeddedPostgres
+import org.flywaydb.core.Flyway
+import java.sql.Connection
+
+class TestDatabase : DatabaseInterface {
+    private val pg: EmbeddedPostgres = try {
+        EmbeddedPostgres.start()
+    } catch (e: Exception) {
+        EmbeddedPostgres.builder().setLocaleConfig("locale", "en_US").start()
```

Comment:
Ikke EU?

Reply:
Copy paste :man_shrugging: Kanskje vi skal prøve å ikke sette den i det hele tatt?

Reply:
Er bare for språk/locale-ting ja - https://www.postgresql.org/docs/current/locale.html. Er jo bare for test-databasen, så kan droppe å ha det med så får vi heller legge det på hvis får problemer med tester.